### ISSUE TITLE
Incorrect breadcrumb links

### ISSUE THREAD

**Issue by monodop** [arduino/reference-en#827]
I am noticing that the breadcrumb links don't seem to be working correctly for me.

For example, on the page: https://www.arduino.cc/reference/en/language/functions/communication/serial/readbytes/

The link to "Serial" in the breadcrumbs points to: https://www.arduino.cc/reference/en#serial

I believe, instead, it should point to: https://www.arduino.cc/reference/en/language/functions/communication/serial/

I've noticed this happen in the past as well, but I don't have all of the examples on hand right now.

![image](https://user-images.githubusercontent.com/9141989/116838566-a7403300-ab83-11eb-9baa-c48640b62f54.png)

**Comment by VeloSteve** [comment]
Still a problem in 2024.  For example, if I am on https://www.arduino.cc/reference/en/language/functions/communication/stream/streamreadbytesuntil/
and I click Stream in the breadcrumb line it takes me to
https://www.arduino.cc/reference/en/
This is just a list of some Arduino functions, and not the Stream information that a user would expect.

Examining this page, there is a Stream link which goes to the correct page (https://www.arduino.cc/reference/en/language/functions/communication/stream/)  but requiring this extra step is a bug.

By the way, it's only a coincidence that I was on a Stream function like the original poster.  This also happens with Wire, Mouse, SPI, and probably all the others.

**Comment by MaderDash** [comment]
> Still a problem in 2024. For example, if I am on https://www.arduino.cc/reference/en/language/functions/communication/stream/streamreadbytesuntil/ and I click Stream in the breadcrumb line it takes me to https://www.arduino.cc/reference/en/ This is just a list of some Arduino functions, and not the Stream information that a user would expect.
> 
> Examining this page, there is a Stream link which goes to the correct page (https://www.arduino.cc/reference/en/language/functions/communication/stream/) but requiring this extra step is a bug.
> 
> By the way, it's only a coincidence that I was on a Stream function like the original poster. This also happens with Wire, Mouse, SPI, and probably all the others.

Just a bump on this, IDK if it's just me or not. But it is working properly for me. Please let me know if this is still an issue.

**Comment by MaderDash** [comment]
> I am noticing that the breadcrumb links don't seem to be working correctly for me.
> 
> For example, on the page: https://www.arduino.cc/reference/en/language/functions/communication/serial/readbytes/
> 
> The link to "Serial" in the breadcrumbs points to: https://www.arduino.cc/reference/en#serial
> 
> I believe, instead, it should point to: https://www.arduino.cc/reference/en/language/functions/communication/serial/
> 
> I've noticed this happen in the past as well, but I don't have all of the examples on hand right now.
> 
> ![image](https://user-images.githubusercontent.com/9141989/116838566-a7403300-ab83-11eb-9baa-c48640b62f54.png)

Same statement. This may be fixed, please check to see if this is still an issue please?

**Comment by per1234** [comment]
@MaderDash the behavior is exactly as described in the bug report for me. When I click the "Serial" link in the breadcrumbs, I am taken to the URL https://www.arduino.cc/reference/en#serial, which is simply the Arduino Language Reference home page (not even scrolled down to the relevant section, since there is no `serial` anchor in the page).

**Comment by monodop** [comment]
@MaderDash I still see the same issue with the breadcrumbs today

**Comment by monodop** [comment]
![serialbreadcrumbs](https://github.com/arduino/reference-en/assets/9141989/9dbc87e1-163c-4aba-9677-37c97f13d98c)

**Comment by VeloSteve** [comment]
Yes, the same issue from 2024 is still there.  The link takes you to the index instead of the topic.
